Zend/zend_exceptions.c

```diff
@@ -740,6 +740,7 @@ void zend_register_default_exception(void) /* {{{ */
 	base_exception_ce->create_object = NULL;
 	memcpy(&default_exception_handlers, zend_get_std_object_handlers(), sizeof(zend_object_handlers));
 	default_exception_handlers.clone_obj = NULL;
+    zend_class_implements(base_exception_ce, 1, zend_ce_throwable);
```

Comment:
Would be great if we could focus on conceptual problems and implementation issues before critiquing whitespace.